### ISSUE TITLE
Does not work in subdirectories

### ISSUE THREAD

**Issue by NobbZ** [oleander/git-fame-rb#56]
given the following folderstructure

```
+ project
  + .git
  + subfolder
```

I can run `git fame` only in `project` while I do get `Error: Branch 'master' does not exist.` in any subfolder. It worked pretty well with 1.7.3 where I got a fame-table of the corresponding subfolder exclusively.

**Comment by oleander** [comment]
In the latest version you need to specify the git directory if you aren't in the same location as the `.git` folder. I'm not sure why I designed it this way, but I remember I had a good reason for it :) You can set the path manually using `--repository="../my/path`.

**Comment by rsharmachamp1** [comment]
@oleander I am still getting the  "Error: Branch 'master' does not exist." even after giving the below
git fame --branch=develop --repository="."

git-fame 2.5.1 (c) 2012-2016 Linus Oleander
Am running on gitBash on windows, gem version 2.5.1

**Comment by oleander** [comment]
@rsharmachamp1 Sorry about that, lets see if we can fix it. Is the `.git` folder located in `.` as you specified?

**Comment by rsharmachamp1** [comment]
yes, its there, and I tried in master as well as other branches too. for all branches it says "<branch_name> does not exist"

**Comment by oleander** [comment]
What does `git branch` yield? Also, could you try to specify the output of `git rev-parse --show-toplevel` as the directory to `--repository=`? So if it yields `/a/b/c` try running `git fame --repository="/a/b/c"`.

**Comment by rsharmachamp1** [comment]
$ git branch
* develop
  master
  milestone_m19

Both of the below give same "Error: Branch 'master' does not exist."
git fame --repository="/C/src/git/rahuls_test"
git fame --repository="C:/src/git/rahuls_test"

I am in C:/src/git/rahuls_test while running all commands, .git directory is also located here.

**Comment by Petah** [comment]
Same issue here. Running windows.

```
C:\work\project>git branch
* develop
  feature/coupons
  feature/orders
  feature/transform

C:\work\project>git-fame --branch=develop .
Error: Branch 'develop' does not exist.
Try --help for help.

C:\work\project>git-fame .
Error: Branch 'master' does not exist.
Try --help for help.
```

**Comment by moyulcy** [comment]
I run git fame in project's folder ,git bash here show  "print(tabulate(auth_stats, stats_tot, args["--sort"]))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 4552-4554: ordinal not in range(128)".

**Comment by Erkebulan69** [comment]
Having the same issue, running in project's folder (where .git is located) displays "Error: Branch 'master' does not exist."

**Comment by grrrrrrrrrrr** [comment]
Same issue, same scenario

**Comment by korthout** [comment]
Same problem here. Also running windows. Using bash through cmder with git version 2.14.2.windows.1

**Comment by kannanbalu** [comment]
Hi, i have exactly the same issue and is blocking my activity on git fame. How do we resolve this? Kindly assist.

**Comment by jackdh** [comment]
I've found a work for windows which is to use Ubuntu Bash windows feature. From there install git_fame and go.
